P7-Api-Endpoints/Authentication/Login.bru

```diff
@@ -0,0 +1,23 @@
+meta {
```

Comment:
This is a remark on the overall folder.

I would suggest moving these files - potentially to a relevant named subfolder of the API solution, as currently it can be confusing its on level with the solutions.